src/main/java/frc/robot/subsystems/AlgeaRemover.java

```diff
@@ -53,17 +54,20 @@ public class AlgeaRemover extends SubsystemBase {
               AlgeaRemoverConstants.kAlgeaP,
               AlgeaRemoverConstants.kAlgeaI,
               AlgeaRemoverConstants.kAlgeaD);
+  private static final SparkMaxConfig algaeMotorConfig = new SparkMaxConfig();
 
   public AlgeaRemover() {
     algeaMotor = new SparkMax(Constants.CANIDs.kAlgeaMotor, MotorType.kBrushless);
+    algaeMotorConfig.smartCurrentLimit(60);
+    algaeMotorConfig.apply(algeaCfg);
     algeaMotor.configure(
-        new SparkMaxConfig().apply(algeaCfg),
-        ResetMode.kNoResetSafeParameters,
-        PersistMode.kNoPersistParameters);
+        algaeMotorConfig, ResetMode.kNoResetSafeParameters, PersistMode.kNoPersistParameters);
     algeaEncoder =
         new DutyCycleEncoder(
             DIOConstants.kAlgeaEncoderID, 1, AlgeaRemoverConstants.encoderOffset.in(Rotations));
-    algeaMotor.getEncoder().setPosition(algeaEncoder.get());
+    algeaMotor
+        .getEncoder()
+        .setPosition(algeaEncoder.get() * AlgeaRemoverConstants.kAlegeaGearRatio);
```

Comment:
There should be a seedEncoder method where this should get updated as well (or possibly instead)